README.md

```diff
@@ -14,34 +14,34 @@ import ENSBuilder from 'ens-builder';
 const builder = new ENSBuilder(wallet);
 ```
 
-The fastest way to setup testing environment is to call `bootstrapWith`:
+The fastest way to start is to use the `bootstrapWith` convenience method:
 ```js
-const provider = await builder.bootstrapWith('example', 'eth');
+const ensAddress = await builder.bootstrapWith('example', 'eth');
 ```
 
-This will setup ENS, register top-level domain `eth` and domain under it `example.eth`.
-It will also create a registrar for `example.eth` as well as a global resolver and a reverse registrar.
+This will setup ENS, register the `eth` top-level domain and the `example.eth` subdomain under it.
+It will also create a registrar for `example.eth`, a reverse registrar, as well as a global resolver.
 
-The method will return `ethers` provider pre-configured with new ENS.
+The method will return the address of the deployed ENS. It can then be used, for example, to setup a new ethers provider and use it's methods for domain resolution:
```

Comment:
I already assumed we will abandon the explicit wrapping with `withENS` as this introduces a number of problems.